napari/_qt/layer_controls/qt_labels_controls.py

```diff
@@ -211,13 +211,20 @@ def __init__(self, layer):
 
         # don't bind with action manager as this would remove "Toggle with {shortcut}"
 
+        self._EDIT_BUTTONS = (
```

Comment:
Better names appreciated :)

Reply:
I think this is fine! 😊

Reply:
If I correctly recognize it, maybe berret name could be `MODE_BUTTONS` as they control layer mode.

Reply:
As it is private it could be changed easily in the future.